### PR TITLE
Add CMD/Ctrl+k

### PR DESCRIPTION
**Pull request by MartinSchoeler**
@RocketChat/core 

Users can now use CMD/Ctrl+k to go to search. Also added a placeholder indicating the shortcut
screenshot from 2017-02-14 15-41-51

### CODE REVIEW

File: packages/rocketchat-ui-sidenav/side-nav/toolbar.js [hunk 1]
```diff
@@ -76,6 +76,17 @@ Template.toolbar.helpers({
...
+	getPlaceholder() {
+		var placeholder = 'Search (Ctrl+K)';
+
+		if (!Meteor.Device.isDesktop()) {
+			placeholder = 'Search';
+		} else if (window.navigator.platform.toLowerCase().includes('mac')) {
+			placeholder = 'Search (CMD+K)';
+		}
+
+		return placeholder;
```
Comment (marceloschmidt) [inline packages/rocketchat-ui-sidenav/side-nav/toolbar.js]: You are not using TAPi18n here.

**Comment by karlprieb** [comment]
I think we need to be able to translate "Search".

The shortcut will appear on mobile?

File: packages/rocketchat-ui-sidenav/side-nav/toolbar.js (hunk 1 above)
Comment (MartinSchoeler) [inline packages/rocketchat-ui-sidenav/side-nav/toolbar.js]: Yes! You are right, totally forgot about that :smile: . Fixed now

**Comment by MartinSchoeler** [comment]
@karlprieb no it wont show on mobile and I fixed the translation now

File: packages/rocketchat-ui-master/master/main.coffee [hunk 2]
```diff
@@ -4,7 +4,7 @@ Template.body.onRendered ->
 	clipboard = new Clipboard('.clipboard')
 
 	$(document.body).on 'keydown', (e) ->
-		if e.keyCode is 80 and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
+		if e.keyCode is 80 or e.keyCode is 75 and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
```
Comment (rodrigok) [inline packages/rocketchat-ui-master/master/main.coffee]: This `if` is not right 😄 
Did you try to hit only the key `p` to see what happens?

The correct code is:
```
if (e.keyCode is 80 or e.keyCode is 75) and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
```
or
```
if e.keyCode in [80, 75] and (e.ctrlKey is true or e.metaKey is true) and e.shiftKey is false
```

Comment (MartinSchoeler) [inline packages/rocketchat-ui-master/master/main.coffee]: Yes you are right! :smile:  Will fix it right away!

**Comment by Rohlik** [comment]
What is difference between `ctrl+p` and `ctrl+k`?

**Comment by sampaiodiego** [comment]
@Rohlik they both act as the same

**Pull request merged**